stixcore/products/level0/quicklook.py

```diff
@@ -49,11 +49,14 @@ def __init__(self, *, service_type, service_subtype, ssid, control, data,
         self.data = data
         self.idb = idb
 
-        self.obs_beg = SCETime.from_float(self.data['time'][0]
-                                          - self.control['integration_time'][0] / 2)
-        self.obs_end = SCETime.from_float(self.data['time'][-1]
-                                          + self.control['integration_time'][-1] / 2)
-        self.obs_avg = self.obs_beg + (self.obs_end - self.obs_beg) / 2
+        try:
+            self.obs_beg = SCETime.from_float(self.data['time'][0]
+                                              - self.control['integration_time'][0] / 2)
+            self.obs_end = SCETime.from_float(self.data['time'][-1]
+                                              + self.control['integration_time'][-1] / 2)
+            self.obs_avg = self.obs_beg + (self.obs_end - self.obs_beg) / 2
+        except ValueError:
+            pass
```

Comment:
Hum if we get a value error here I think we are in bad shape do you know what was causing it?

Reply:
yes this was about my question about the timing conversions
some old test fits file had SCETime other "real" time within